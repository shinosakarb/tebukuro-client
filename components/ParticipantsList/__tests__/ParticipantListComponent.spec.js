import { shallow } from 'enzyme'
import shallowToJson from 'enzyme-to-json'
import ParticipantList from '../index'
import ParticipantParams from '../../../factories/Participant'

describe('ParticipantList Component', () => {
  it('renders self.', () => {
    const participants = Object.values(ParticipantParams)
    const component = shallow(<ParticipantList participants={participants} />)
    const tree = shallowToJson(component)

    expect(tree).toMatchSnapshot()
  })
})
