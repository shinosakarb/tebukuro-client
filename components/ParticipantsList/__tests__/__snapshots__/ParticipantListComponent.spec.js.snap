// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ParticipantList Component renders self. 1`] = `
<div>
  <ul>
    <li
      key="1"
    >
      <Participant
        eventId={1}
        id={1}
        name="participant1"
      />
    </li>
    <li
      key="2"
    >
      <Participant
        eventId={1}
        id={2}
        name="participant2"
      />
    </li>
  </ul>
</div>
`;
